src/scheduler.py

```diff
@@ -80,7 +80,18 @@ def _run_job(self, job_config, runtime, platform, input_node):
         data = runtime.generate(job, params)
         tmp = tempfile.TemporaryDirectory(dir=self._output)
         output_file = runtime.save_file(data, tmp.name, params)
-        running_job = runtime.submit(output_file)
+        try:
+            running_job = runtime.submit(output_file)
+        except Exception as e:
+            self.log.error(' '.join([
```

Comment:
It would probably be neater to collect node/runtime/job data first and append it with TestJob ID/error message for logging but it could be the next step